apps/src/constants.js

```diff
@@ -44,6 +44,7 @@ exports.TestResults = {
 
   // start using negative values, since we consider >= 20 to be "solved"
   EMPTY_FUNCTION_NAME: -1,       // We have a variable or function with the name ""
+  NESTED_FOR_SAME_VARIABLE: -2,
```

Comment:
I didn't actually validate, but I assume so. The danger might be if we somehow end up in a case where we end up with NO_TESTS_RUN and give an error message about EMPTY_FUNCTION_NAME. Not sure how big a risk this is/if it makes sense to change at this point. What do you think?